cradle-cassandra/build.gradle

```diff
@@ -8,11 +8,13 @@ configurations.all {
 }
 
 dependencies {
-	compile project(":cradle-core")
-	compile "com.datastax.oss:java-driver-core:${driver_version}"
-	compile "com.datastax.oss:java-driver-query-builder:${driver_version}"
-	compile "com.datastax.oss:java-driver-mapper-processor:${driver_version}"
-	compile "com.datastax.oss:java-driver-mapper-runtime:${driver_version}"
+	implementation project(":cradle-core")
+	implementation "commons-io:commons-io:2.5"
+	implementation "org.apache.commons:commons-lang3:3.8.1"
+	implementation "com.datastax.oss:java-driver-core:${driver_version}"
+	implementation "com.datastax.oss:java-driver-query-builder:${driver_version}"
+	implementation "com.datastax.oss:java-driver-mapper-processor:${driver_version}"
+	implementation "com.datastax.oss:java-driver-mapper-runtime:${driver_version}"

```

Comment:
There are two configurations instead of `compile` one:

1. implementation - the dependencies that are required to run library in runtime but not exposed via public methods
2. api - the dependencies that are required to use library (exposed by public methods)


So, I think the `project(":cradle-core")` and `com.datastax.oss:java-driver-query-builder` should be api dependencies. You can create a simple project and try to use cradle library. If some dependencies are missing than they should be marked as `api`

More information about `api` and `implementation` can be found here: https://docs.gradle.org/current/userguide/java_library_plugin.html#sec:java_library_separation